lib/vector-config-macros/src/configurable_component.rs

```diff
@@ -342,8 +342,9 @@ pub fn configurable_component_impl(args: TokenStream, item: TokenStream) -> Toke
         .typed_component()
         .map(|tc| tc.get_component_desc_registration(&input));
 
-    // Generate and apply all of the necessary derives.
+    // Generate and apply all of the necessary derives
```

Comment:
**question (non-blocking)**: Thought we used full punctuation in these?

Reply:
Oh yeah, fair. This is because I removed some new comments I had added as part of the auto-`deny_unknown_fields` stuff which removed the period, but I removed the code by hand rather than reverting it.

I'll clean this up.

Reply:
Fixed in 7b8fe95.